renovate.json5

```diff
@@ -2,7 +2,7 @@
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
 	"extends": ["config:base"],
 	"automergeType": "branch",
-	"dependencyDashboard": true,
+	"dependencyDashboard": false,
```

Comment:
@olyop this is a surprising change hidden away in a PR about ESM. Also I changed this to true awhile ago and you approved it? I still think this is a good thing to have, but not a huge deal.